account_invoice_show_currency_rate/README.rst

```diff
@@ -0,0 +1,85 @@
+==================================
+Account Invoice Show Currency Rate
+==================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-invoicing/tree/13.0/account_invoice_show_currency_rate
+    :alt: OCA/account-invoicing
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-invoicing-13-0/account-invoicing-13-0-account_invoice_show_currency_rate
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/95/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module show currency rate applied in invoices.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+o use this module, the company have to access rights Multi Currencies.
+
+#. Go to Invoicing > Configuration > Settings > Currencies > Multi-Currencies
```

Comment:
This would probably fit better under `Configuration`. Also, maybe it could be clearer like this:
```suggestion
To use this module, the company must have Multi Currencies enabled.

#. Go to Invoicing > Configuration > Settings > Currencies > Multi-Currencies
```